### ISSUE TITLE
Checkbox icons not appearing

### ISSUE THREAD

**Issue by jkole** [lumapps/lumX#374]
When using the checkbox classes, the mdi icons don't appear for either state (checked or unchecked) because it seems the glyph is not being found (shows a thick square symbol on Chrome, blank on Safari).  The pseudo :after elements on .checkbox__label appear to be getting the correct hex codes (f203 for unchecked, f204 for checked) and these same icons appear perfectly when I use the .mdi class directly as follows:

```
<i class='mdi mdi-checkbox-blank-outline'></i>
<i class='mdi mdi-checkbox-marked'></i>
```

Markup for the checkbox that is not working:

```
<div class="checkbox">
  <input type="checkbox" id="remember-checkbox" class="checkbox__input" ng-model="ctrl.remember">
    <label for="remember-checkbox" class="checkbox__label">
      Keep me signed in
    </label>
</div>
```

Can't for the life of me figure this one out...

**Comment by AntiFreez** [comment]
It was broken in the latest release of MaterialDesign-Webfont: font name was changed from "MaterialDesignIcons" to "Material Design Icons".
https://github.com/Templarian/MaterialDesign-Webfont/commit/13aa4432f3ea05b27bc3d073473f5dc28f376f83

**Comment by AntiFreez** [comment]
It is necessary to fix mdi mixin: 
``` scss
@mixin mdi() {
    display: inline-block;
    font: normal normal normal #{$mdi-font-size-base}/1 MaterialDesignIcons;
    font-size: inherit;
    text-rendering: auto;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
    transform: translate(0, 0);
}
```

**Comment by AntiFreez** [comment]
See https://github.com/lumapps/lumX/pull/370

**Comment by matmkian** [comment]
We've just released LumX v1.0.0rc1. This new version fixes the issue, so I'm closing it.
